web-registry/src/App.tsx

```diff
@@ -103,14 +103,30 @@ Sentry.init({
 
 export const history = createBrowserHistory();
 
-const GoogleAnalytics: React.FC = (): JSX.Element => {
-  useGoogleAnalyticsInit();
+const PageViewTracking: React.FC = (): JSX.Element => {
+  const location = useLocation();
+  const analytics = useAnalytics();
+  useEffect(() => {
+    // send page view whenever react-router location changes
+    // this sends page views to all analytics plugins:
+    // https://getanalytics.io/plugins/
+    analytics.page({
+      path: location.pathname,
+      search: location.search,
+      title: location.pathname,
+    });
+  }, [location, analytics]);
   return <></>;
 };
 
 const App: React.FC = (): JSX.Element => {
   const { user, isLoading, error } = useAuth0();
   const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
+  const { plugins } = useAnalytics();
+  // user opt-in, right now we just enable all of the analytics
+  // providers. but this shows how to conditional enable or even
+  // disable certain providers.
+  plugins.enable(['amplitude', 'google-analytics']).then(() => {});
```

Comment:
this is the function to enable analytics plugins. if/when we implement user opt-in/opt-out in the UI, we can enable/disable using code similar to this. right now, we just enable all of the analytics plugins.